manager/integration/tests/test_infra.py

```diff
@@ -0,0 +1,129 @@
+import os
+import pytest
+import time
+
+from digitalocean import digitalocean
+from common import get_core_api_client, get_longhorn_api_client
+from common import RETRY_COUNTS, RETRY_INTERVAL
+
+
+def detect_cloudprovider():
+    cloudprovider_name = os.getenv("CLOUDPROVIDER")
+
+    if cloudprovider_name == "digitalocean":
+        cloudprovider = digitalocean()
+        return cloudprovider
+
```

Comment:
`assert False` if we cannot find a suitable cloud provider.

Reply:
done